File engine documentation

MassiminoilTrace opened the pull request:
References issue #101 

Let me know if you have any suggestion or improvement.

Needs a check from a native English speaker probably.

Demonthos commented:
I checked the english on what you have so far. I see this PR is still a draft. Whenever you are finished, open the PR and I can review it

MassiminoilTrace commented:
Thank you
The `directory` attribute to select a folder path didn't seem to work when I first tried it, that's why I added the "webkitdirectory" manually. Thanks for the correction.

I changed "multiple files" to "directory" in 74e4ef8166 because I guess that's the reason the directory attribute exists in dioxus. A standardized way to select multiple files in HTML already exists [mdn docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/multiple), but the directory path selection doesn't exists [mdn docs webkitdirectory](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#webkitdirectory) .

ealmloff reviewed (approved):
Looks great, Thank you!
